### PR TITLE
Hotfix MeLink G2L

### PR DESCRIPTION
**Pull request by mistyron**
Pull request for changes applied which include:
1. Only check permissions of cache directories (inside contyainer), when they are available
2. Assign 777 permissions to cache directory upon creation

### CODE REVIEW

File: Build/exec.sh [hunk 1]
```diff
@@ -15,7 +20,10 @@ then
 	echo "Unable to obtain full acess  permissions to 'output' and its sub directories, edit the permissions of 'output' accordingly! exit"
 	exit -1
 fi
-if [[ ! -w $WORK/build/sstate-cache || ! -w $WORK/build/downloads ]];
+echo "WORK ${WORK}"
+#if [[ ! -w $WORK/build/sstate-cache || ! -w $WORK/build/downloads ]];
```
Comment (matinlotfali) [inline Build/exec.sh:24]: Don't need the above two lines

File: Build/exec.sh [hunk 2]
```diff
@@ -15,7 +20,10 @@ then
...
+if [[ -d $WORK/build/sstate-cache && -d $WORK/build/downloads && (! -w $WORK/build/sstate-cache || ! -w $WORK/build/downloads) ]];
+
```
Comment (matinlotfali) [inline Build/exec.sh:26]: Don't need the extra empty line

File: Build/run.sh [hunk 3]
```diff
@@ -64,7 +64,7 @@ then
   /usr/bin/docker run --privileged ${USE_TTY} --rm -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out --name ${CONTNAME} ${CONTNAME}
 else
 	#Create CPATH sub directories if they do not exist
-	mkdir -p ${CPATH}/downloads
-	mkdir -p ${CPATH}/sstate-cache/${MPU}
+	mkdir -p -m777 ${CPATH}/downloads
+	mkdir -p -m777 ${CPATH}/sstate-cache/${MPU}
 	/usr/bin/docker run --privileged ${USE_TTY} --rm -v "${PWD}/${OUTDIR}":/home/yocto/rzg_vlp_v3.0.0/out -v "${CPATH}/downloads":/home/yocto/rzg_vlp_v3.0.0/build/downloads -v "${CPATH}/sstate-cache/${MPU}/":/home/yocto/rzg_vlp_v3.0.0/build/sstate-cache -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} --name ${CONTNAME} ${CONTNAME}
```
Comment (matinlotfali) [inline Build/run.sh:69]: Should probably remove all references to ${DLOAD} in this file

**Review by matinlotfali — CHANGES_REQUESTED** [review]

File: Build/exec.sh (hunk 2 above)
Comment (mistyron) [reply Build/exec.sh:26]: removed

File: Build/run.sh (hunk 3 above)
Comment (mistyron) [reply Build/run.sh]: was just a draft but i've cleaned it up now!

**Review by matinlotfali — APPROVED** [review]
LGTM

**Pull request merged**